FireRender.Maya.Src/Context/FireRenderContext.cpp

```diff
@@ -809,29 +809,29 @@ void FireRenderContext::setupDenoiserFB()
 void FireRenderContext::UpdateDefaultLights()
 {
 	MAIN_THREAD_ONLY;
-	bool enabled = true;
+	bool airVolumeEnabled = true;
```

Comment:
why do we have  these changes in function UpdateDefaultLights ?

Reply:
mistake. fixed